Update: ETR change may need more than 30min (#1227)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/security-and-compliance/mail-flow-rules/manage-mail-flow-rules.md

You can create a mail flow rule by setting up a Data Loss Prevention (DLP) policy, creating a new rule, or by copying a rule. You can use the Exchange admin center (EAC) or Exchange Online PowerShell.

> [!NOTE]

> > After you create or modify a mail flow rule, it can take up to 30 minutes and more in some case for the new or updated rule to be applied to email.

Reviewer: ```suggestion
> After you've created or modified a mail flow rule, it can take up to 30 minutes, and even more in some cases, for the new or updated rule to be applied to emails.
```

## View or modify a mail flow rule

> [!NOTE]

> > After you create or modify a mail flow rule, it can take up to 30 minutes and more in some case for the new or updated rule to be applied to email.

Reviewer: ```suggestion
> After you've created or modified a mail flow rule, it can take up to 30 minutes, and even longer in some cases, for the new or updated rule to be applied to emails.
```
